packages/modules/manage-versions/src/mocks.ts

```diff
@@ -48,6 +48,7 @@ export const MockedChangeset = (
     displayName: `${index}`,
     description: `ch_description${index}`,
     pushDateTime: MOCKED_DATE,
+    synchronizationInfo: { changedFiles: ["test-file-${index}.dgn"] },
```

Comment:
You should use ` not "

Reply:
These errors didn't show up for me, pushed the change though